exercise/async_views.py

```diff
@@ -52,6 +52,12 @@ def _post_async_submission(request, exercise, submission, errors=None):
         submission.feedback = form.cleaned_data["feedback"]
         submission.grading_data = post_data
 
+        # Acos-submissions initialize this as an empty string for some reason; fix here
```

Comment:
The `Submission.meta_data` field uses our old custom `JSONField` that is simply a normal `TextField` with some extra JSON parsing. Thus, its default value is an empty string. In the case of Acos exercises, no value is set to the `submission.meta_data` field before this point. The submission is created here:

https://github.com/apluslms/a-plus/blob/b36e145cbb91b5a3b47204c122b081571d1bcee8/exercise/api/views.py#L128